.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,44 @@
+name: Build And Deploy
+
+on:
+  push:
+    branches: ["main"]
+env:
+  DAY_OF_WEEK: Monday
+  APP_KEYS:value_1,value_2,value_3
+  API_TOKEN_SALT:API_TOKEN_SALT_VALUE
+  ADMIN_JWT_SECRET:ADMIN_JWT_SECRET_VALUE
+  JWT_SECRET:__SKIP_FOR_LOCAL_ENVIRONMENT--GENERATED__
+jobs:
```

Comment:
These should be accessible and not need renamed

Reply:
not sure what you mean here?

Reply:
No point in doing the mapping here is what I'm saying, you can just use the values directly